addons/fools/addon.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "eastah egg",
+  "description": "o_0",
+  "tags": ["community"],
+  "enabledByDefault": false,
```

Comment:
We'll probably end up doing some hardcoded logic for April Fools day like other years, right?